sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java

```diff
@@ -96,14 +96,6 @@ public final class QueueDescription {
             namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
     private Integer maxDeliveryCount;
 
-    /*
-     * The maximum message size for a message.
-     */
-    @JacksonXmlProperty(
-        localName = "MaxMessageSizeInKilobytes",
-        namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
-    private Long maxMessageSizeInKilobytes;
-
     /*
```

Comment:
The `QueueDescription` is generated by swagger, here is the [README ](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/servicebus/azure-messaging-servicebus/swagger)for how to generate.

However, I find the swagger.json file is out of date and we done some manually changes on the generated models (`maxMessageSizeInKilobytes` is one of them).  I created #32149 to keep updating our swagger. Before that, I think you can try to update `MaxMessageSizeInKilobytes` field in swagger file, regenerate and ignore other changes.